resources/developer-guides/daemon-rpc.md

```diff
@@ -108,6 +107,7 @@ Outputs:
 
 * *count* - unsigned int; Number of blocks in longest chain seen by the node.
 * *status* - string; General RPC error code. "OK" means everything looks good.
+* *untrusted* - boolean; States if the result is obtained using the bootstrap mode, and is therefore not trusted (`true`), or when the daemon is fully synced (`false`).
```

Comment:
The second part is confusing. Took me a bit to realize it was correct :) Maybe: "or when the daemon is fully synced and thus handles the RPC locally" ?

Reply:
That's copied from the definition of *unstrusted* in another call. I can change it everywhere, no problem.